File: python/1159.py
while True:
    
    x = int(input())
    
    if x == 0:
        break
    
    count = 0
    g = x
    sum = 0
    
    while count < 5:
        if g % 2 == 0:
            sum += g
            count += 1
        g += 1
        
    print(sum)

File: lua/1159.lua
while true do
    local x = tonumber(io.read())
    if x == 0 then
        break
    end
    local sum = 0
    for i=x, x+9 do
        if i % 2 == 0 then
            sum = sum + i
        end
    end
    print(sum)
end